core/run/src/lib.rs

```diff
@@ -682,6 +684,14 @@ impl Axon {
         Ok(())
     }
 
+    fn run_ibc() {
```

Comment:
The method should be provided in `core-ibc`, and we only need to spawn the future like this
```suggestion
    tokio::spawn(async {
        run_ibc_grpc(port).await;
    }).await;
```